src/components/Dissolution/AssociationDetails.vue

```diff
@@ -161,12 +161,12 @@ export default class AssociationDetails extends Mixins(CommonMixin, EnumMixin, D
   }
 
   /** Whether the address object is empty. */
-  private isEmptyAddress (address: AddressIF): boolean {
+  protected isEmptyAddress (address: AddressIF): boolean {
```

Comment:
`isEmptyAddress()` needs to be protected so the template (HTML) can call it. (This is not yet enforced by the framework but this is correct and the previous code was incorrect.)